community/modules/compute/pbspro-execution/main.tf

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  resource_prefix = var.name_prefix != null ? var.name_prefix : "${var.deployment_name}-exec"
+  # gather local (inside VM) mountpoints with trailing slashes
+  mountpoints = [for ns in var.network_storage : endswith(ns.local_mount, "/") ? ns.local_mount : "${ns.local_mount}/"]
```

Comment:
nit: It is not clear where the trailing slash requirement is coming from. Since you are already adding a comment it would be great to leave a breadcrumb.

Reply:
Resolved on HEAD of PR. Your comment also alerted me to another instance of `endswith` which was introduced in Terraform 1.3 (I'm surprised it's so new). I've replaced that with a Terraform 1.0-friendly solution.